docs/updating_schemas.md

```diff
@@ -102,7 +117,7 @@ Refer to [object-lifecycle.md](https://github.com/IGVF-DACC/igvfd/blob/dev/docs/
                 schema = load_schema('encoded:schemas/treatment.json')
 
 
-5. Within in a class add in  *embedding*, *reverse links*, and *calculated properties* as necessary.
+6. Within in a class add in  *embedding*, *reverse links*, and *calculated properties* as necessary.
```

Comment:
`Within in` -> `Within`

Reply:
There's also slim embedding using `embedded_with_frame`.

Reply:
@keenangraham now that we have slim_embedding should we ever use embedding?

Reply:
Still kind of think `embedding` should be the default since that was the original intention of the system, though maybe it doesn't matter. I don't fully understand the consequences of using slim embedding everywhere in the big context of our system (e.g. JSON-LD). I know the Opensearch mapping code still assumes all of the fields are embedded from another object, but maybe that's something that can just be changed. The original intention was to break circular dependencies and avoid expensive calculated properties (e.g. an experiment that wants some information from a reference dataset that embeds the experiment itself and 1000 other experiments). I'm not sure if that means it should be used everywhere.